[DOCS] Update case priv docs (#1443)
elastic/security-docs: docs/getting-started/cases-req.asciidoc

NOTE: To send cases to external systems, you need the
https://www.elastic.co/subscriptions[appropriate license].

> To grant access to cases, set the {kib} space privileges for the *Cases* and *Actions and Connectors* features as follows:

Reviewer: do we need to say that `Actions and Connectors` privileges are just here if you want to push a case to an external service like `serviceNow`?

> To grant access to cases, set the {kib} space privileges for *Cases* and *Actions and Connectors* feature. More information about the minimum privileges needed to use cases is below:

Reviewer 2: ```suggestion
To grant access to cases, set the {kib} space privileges for the *Cases* and *Actions and Connectors* features. More information about the minimum privileges needed to use cases is below:
```

> To grant access to cases, set the {kib} space privileges for the *Cases* and *Actions and Connectors* features. More information about the minimum privileges needed to use cases is below:

Reviewer 3: ```suggestion
To grant access to cases, set the {kib} space privileges for the *Cases* and *Actions and Connectors* features as follows:
```
Feel like we could cut out some words here.

Author: This is more succinct and more accurate to what the table portrays. The table describes how to set varying levels of access, not just the minimum privs.
